src/app/components/WorkDetail/WorkHeader.jsx

```diff
@@ -1,31 +1,34 @@
 import React from 'react';
```

Comment:
This component is throwing an error in my console saying that a list is missing a unique key. I can't see where its coming from though.

Reply:
Yeah I see it too - I think it's from something in the Design System.  I'm not gonna deal with it yet, it seems like a rabbit hole.

Reply:
For sure, I don't mind the warnings just feel like it should be on a list to address at some point